anta/tests/cvx.py

```diff
@@ -87,3 +89,94 @@ def test(self) -> None:
         cluster_status = command_output["clusterStatus"]
         if (cluster_state := cluster_status.get("enabled")) != self.inputs.enabled:
             self.result.is_failure(f"Management CVX status is not valid: {cluster_state}")
+
+
+class VerifyCVXClusterStatus(AntaTest):
+    """Verifies the CVX Server Cluster status.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if all of the following conditions is met:
+        - CVX Enabled state is true   
+        - Cluster Mode is true
+        - Role is either Master or Standby
```

Comment:
The test as written is just checking that device role == input role. So technically if the command output says 
"Disconnected" and the input says "Disconnected" the test will pass but the success condition will not be met. 

Should we change the logic or the condition?
```suggestion
        - Role is either Master or Standby.
```